src/pages/library/DataBrowserPreview.js

```diff
@@ -129,11 +139,30 @@ const DataBrowserPreview = ({ id }) => {
     libraryTopMatter(activeTab),
     !snapshot ?
       centeredSpinner() :
-      div({ style: { padding: 20 } }, [
+      div({ style: { padding: 20, display: 'flex', flexDirection: 'column', height: '100%' } }, [
         div({ style: { display: 'flex', flexDirection: 'row', alignItems: 'top', width: '100%', lineHeight: '26px' } }, [
           h1([snapshot['dct:title']])
         ]),
-        div({ style: { display: 'flex', flexDirection: 'row', alignItems: 'center', width: '100%', marginBottom: 30 } }, [
+        snapshot.access === snapshotAccessTypes.CONTROLLED && div({ style: { display: 'flex', flexDirection: 'row', backgroundColor: 'white', fontSize: '1.1rem', lineHeight: '1.7rem', padding: '20px 30px 25px', width: 'fit-content', margin: 'auto' } }, [
+          icon('lock', { size: 27, style: { marginRight: 20, marginTop: 5, color: commonStyles.access.controlled } }),
```

Comment:
Given this morning's conversation, will this be getting removed?

Reply:
yes!